modules/EC2/variable.tf

```diff
@@ -0,0 +1,35 @@
+variable "identifier" {
+    type = string
+    default = "shivani"
+  
+}
+
+variable "ami_id" {
+    type = string
+    default = "ami-08a52ddb321b32a8c"
```

Comment:
AMI ID is something that changes between regions. To make sure these templates run in every AWS Region you can make use of a "data" statement to retrieve the AMI ID based on some parameters such as the AMI Name, the OS, the arch, etc
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami.html